File: python/__default.py
from main.basic_unit_tests import basic_unit_tests
test = basic_unit_tests()

# DESC
# Make a function that adds two numbers
# --------------- #


def add(n1, n2):
    return n1+n2


# --------------- #
# TEST
test.describe("Addition")
test.assert_equals(add(2, 2), 4)

File: javascript/__default.js
const basic_unit_tests = require('./main/basic_unit_tests');
let Test = new basic_unit_tests();

// DESC
/*
Make a function that adds two numbers
*/
/* --------------- */

function add(n1, n2) {
    return n1+n2;
}

/* --------------- */
// TEST
Test.describe("Addition");
Test.assertEquals(add(2, 2), 4);